frontend/apps/reader/modules/readerrolling.lua

```diff
@@ -1087,9 +1102,7 @@ function ReaderRolling:updatePos(force)
     -- Allow for the new rendering to be shown before possibly showing
     -- the "Styles have changed..." ConfirmBox so the user can decide
     -- if it is really needed
-    UIManager:scheduleIn(0.1, function ()
-        self:onCheckDomStyleCoherence()
-    end)
+    UIManager:scheduleIn(0.1, self.onCheckDomStyleCoherence, self)
```

Comment:
I prefer the readability of the one with the function() wrapper - and for consistency with the others that may do just a bit more, like in your next chunk.

Reply:
It usually depends on my mood, but avoiding a closure here means we can easily unschedule it without storing a reference to the scheduled closure, so it's very much on purpose there.

(I'm not necessarily a fan of doing the guarding *inside* the closure like some of them do, for instance).